packages/@coorpacademy-components/src/atom/autocomplete/index.js

```diff
@@ -22,17 +21,22 @@ const Autocomplete = props => {
     onFetch = noop,
     onClear = noop,
     onBlur = noop,
-    onSuggestionSelected = noop
+    onSuggestionSelected = noop,
+    title: propsTitle
   } = props;
 
-  const title = `${props.title}${required ? ' *' : ''}`;
+  const title = `${propsTitle}${required ? ' *' : ''}`;
   const className = getClassState(style.default, style.modified, style.error, modified, error);
 
-  const handleChange = e => onChange(e);
-  const handleBlur = (e, selectedSuggestion) => onBlur(e, selectedSuggestion);
-  const handleSuggestionsFetchRequested = e => onFetch(e);
-  const handleSuggestionsClearRequested = e => onClear(e);
-  const handleSuggestionsSelected = (e, data) => onSuggestionSelected(data);
+  const handleChange = useMemo(() => e => onChange(e), [onChange]);
+  const handleBlur = useMemo(() => (e, selectedSuggestion) => onBlur(e, selectedSuggestion), [
+    onBlur
+  ]);
+  const handleSuggestionsFetchRequested = useMemo(() => e => onFetch(e), [onFetch]);
+  const handleSuggestionsClearRequested = useMemo(() => e => onClear(e), [onClear]);
+  const handleSuggestionsSelected = useMemo(() => (e, data) => onSuggestionSelected(data), [
+    onSuggestionSelected
+  ]);
```

Comment:
```suggestion
  const handleSuggestionsSelected = useMemo(() => (e, data) => onSuggestionSelected(data), [
    onSuggestionSelected,
    data
  ]);
```

Applicable everywhere, and please install https://www.npmjs.com/package/eslint-plugin-react-hooks so we get warned

Reply:
`data` is not in this scope. You misread the code.

Yeah, we could add `eslint-plugin-react-hooks`. You could do a PR on https://github.com/CoorpAcademy/eslint-plugin-coorpacademy for that.

Reply:
Oh yea